### PR TITLE
Added cloudi.1.7.3

### PR DESCRIPTION
**Pull request by okeuday**

### CODE REVIEW

**Comment by okeuday** [comment]
The opam-lint warning was fixed.  The build failures appear to be due to the build system.

**Comment by rgrinberg** [comment]
@okeuday You should consider using https://github.com/ocaml/dune/ as your build system. Manually creating makefiles that are portable and correct is almost impossible

File: packages/cloudi/cloudi.1.7.3/opam [hunk 1]
```diff
@@ -0,0 +1,18 @@
+opam-version: "1.2"
+name: "cloudi"
+version: "1.7.3"
+maintainer: "Michael Truog <mjtruog@gmail.com>"
+authors: "Michael Truog <mjtruog@gmail.com>"
+homepage: "http://cloudi.org/"
+bug-reports: "https://github.com/CloudI/CloudI/issues"
```
Comment (jpdeplaix) [inline packages/cloudi/cloudi.1.7.3/opam]: Isn't it more accurate to give the issue tracker for ```cloudi_api_ocaml``` instead ?

**Comment by jpdeplaix** [comment]
The warning here tells the truth. The remove field is unnecessary because you have a .install file that is already taking care of removing automatically the installed files.
Could you do this change and I'll be happy to merge this PR ?

File: packages/cloudi/cloudi.1.7.3/opam (hunk 1 above)
Comment (okeuday) [reply packages/cloudi/cloudi.1.7.3/opam]: @jpdeplaix All the source code is developed and tested in the `https://github.com/CloudI/CloudI/` repository, so I would prefer to have the issues there.  The separate `https://github.com/CloudI/cloudi_api_*` repositories are for development specific to creating programming language packages for the CloudI API.

**Comment by okeuday** [comment]
@jpdeplaix Ok, thanks for pointing that out.  I didn't realize the .install file made the remove field unnecessary.  I have updated the opam file, so it should be ready now.

**Comment by jpdeplaix** [comment]
Awesome. Thanks a lot !

**Pull request merged**